tests/test_venue_request_v2.py

```diff
@@ -3264,27 +3270,20 @@ def test_supplementary_material_revision(self, client, test_client, selenium, re
 
         revision_invitation = openreview.tools.get_invitation(openreview_client, 'V2.cc/2030/Conference/Submission1/-/Supplementary_Material')
         assert revision_invitation
+        assert revision_invitation.edit['readers'] == ['everyone']
         revision_invitation = openreview.tools.get_invitation(openreview_client, 'V2.cc/2030/Conference/Submission2/-/Supplementary_Material')
-        assert revision_invitation
+        assert revision_invitation.edit['readers'] == ['V2.cc/2030/Conference',
+            'V2.cc/2030/Conference/Submission2/Senior_Area_Chairs',
+            'V2.cc/2030/Conference/Submission2/Area_Chairs',
+            'V2.cc/2030/Conference/Submission2/Reviewers',
+            'V2.cc/2030/Conference/Submission2/Authors']
         revision_invitation = openreview.tools.get_invitation(openreview_client, 'V2.cc/2030/Conference/Submission3/-/Supplementary_Material')
         assert revision_invitation
+        assert revision_invitation.edit['readers'] == ['V2.cc/2030/Conference',
+            'V2.cc/2030/Conference/Submission3/Senior_Area_Chairs',
+            'V2.cc/2030/Conference/Submission3/Area_Chairs',
+            'V2.cc/2030/Conference/Submission3/Reviewers',
+            'V2.cc/2030/Conference/Submission3/Authors']
 
         assert all(x not in revision_invitation.edit['note']['content'] for x in ['title','authors', 'authorids','abstract','keywords', 'TLDR'])
-        assert 'supplementary_material' in revision_invitation.edit['note']['content']
-
-#         #make sure homepage webfield was not overwritten after doing get_conference()
```

Comment:
are the tests failing here?

Reply:
I think we copied this over from `test_venue_request.py` when we starting creating API2 tests. I don't think it applies to this test but I can try to adapt it.

Reply:
let's remove the commented code then

Reply:
yep I removed it in this PR already